tests/vmi_multus_test.go

```diff
@@ -1235,3 +1199,22 @@ func checkSriovEnabled(virtClient kubecli.KubevirtClient, sriovResourceName stri
 	}
 	return false
 }
+
+func validateSriovNetworks(virtClient kubecli.KubevirtClient, vmiPod *k8sv1.Pod, networkName, sriovResourceName string) error {
+	out, err := tests.ExecuteCommandOnPod(
+		virtClient,
+		vmiPod,
+		"compute",
+		[]string{"sh", "-c", fmt.Sprintf("echo $KUBEVIRT_RESOURCE_NAME_%s", networkName)},
+	)
+	if err != nil {
+		return err
+	}
+
+	expectedSriovResourceName := fmt.Sprintf("%s\n", sriovResourceName)
+	if out != expectedSriovResourceName {
+		return fmt.Errorf("expected network name %s didnt match actual env settings %s", expectedSriovResourceName, out)
```

Comment:
i cant remove it from the variable, because there is already `sriovResourceName`
unless i TrimSuffix and then i just compare to it.
i will remove meanwhile from the error string

Reply:
Used TrimSuffix and now there isnt expected anymore